src/shared/clients/codecatalystClient.ts

```diff
@@ -38,24 +38,33 @@ interface CodeCatalystConfig {
     readonly gitHostname: string
 }
 
-export function getCodeCatalystConfig(): CodeCatalystConfig {
-    const stage = DevSettings.instance.get('cawsStage', 'prod')
```

Comment:
Can we keep `cawsStage` for backwards compatibility, but in the case `codeCatalystService` exists then that takes priority?

Reply:
I agree with backwards compatibility in general but I am wondering if it is needed at this level? `cawsStage` seems used for only very specific stages of development and I imagine it isn't widely used . Would like to keep things cleaner by avoiding redundant settings if the cost is low.

Reply:
yeah we don't need back-compat for develper settings (usually)